netbox/dcim/models/cables.py

```diff
@@ -317,10 +317,14 @@ def clean(self):
         super().clean()
 
         # Check for existing termination
-        existing_termination = CableTermination.objects.exclude(cable=self.cable).filter(
+        qs = existing_termination = CableTermination.objects.filter(
             termination_type=self.termination_type,
             termination_id=self.termination_id
-        ).first()
+        )
+        if self.cable.pk:
+            qs = qs.exclude(cable=self.cable)
+
+        existing_termination = qs.first()
```

Comment:
Is this change related to supporting Django 5.0? (Also, `existing_termination` is declared twice.)

Reply:
fixed duplicate assignment. This is from this issue this issue https://forum.djangoproject.com/t/unsaved-model-instances-to-related-filters-in-tests-for-django-5-0/26586/2 from "Passing unsaved model instances to related filters is no longer allowed."